readme.md

```diff
@@ -1,22 +1,42 @@
+I felt a little overwhelmed by the description.
+It just jumped right into this directed graph stuff without justification, as the reader, I felt lost and like I didn't know what we were doing.
+
+Maybe something like this.
+
 ## Description
-Most Mandarin words are two characters long. Because of this characters can be viewed as nodes in a directed graph with words defining the edges. Learning vocabulary can then be understood as exploring this directed graph. One learning strategy is to start at the most frequently used character and add new characters to your vocabulary in order of frequency. Each time a new character is learned the words connecting it to the previously known subgraph are also learned. 
+Most Mandarin words are two characters long. There are about 4000 characters. A typical student will learn the meaning of each character, and then start learning words, which are combinations of these characters. This can be a daunting task - you must first learn 4000 characters (or maybe you optimize and only learn some subset of the 4000) before you start learning words. The mandarin learner is constantly confronted with the following question: what characters should I next learn? It turns out that a judicious choice of which characters to learn next can greatly smooth the learning process.
+
+With this in mind, learning Mandarin can be cast as exploring a directed graph, where each character is a node, and each edge is a word. 
+
+(INSERT SUPER COOL DIRECTED GRAPH, in order to help the reader picture what you are picturing)
```

Comment:
```
![image info](./pictures/image.png)
```

Reply:
https://stackoverflow.com/questions/41604263/how-do-i-display-local-image-in-markdown